packages/catala/catala.0.3.0/opam

```diff
@@ -16,7 +16,7 @@ depends: [
   "menhir" {>= "20200211"}
```

Comment:
```suggestion
  "gen" {>= "0.2.3"}
  "menhir" {>= "20200211"}
```
To fix
```
#       ocamlc src/catala/surface/.surface.objs/byte/surface__Parser_driver.{cmo,cmt} (exit 2)
# (cd _build/default && /home/opam/.opam/4.08/bin/ocamlc.opt -w -40 -g -bin-annot -I src/catala/surface/.surface.objs/byte -I /home/opam/.opam/4.08/lib/ANSITerminal -I /home/opam/.opam/4.08/lib/bindlib -I /home/opam/.opam/4.08/lib/bytes -I /home/opam/.opam/4.08/lib/calendar -I /home/opam/.opam/4.08/lib/camomile -I /home/opam/.opam/4.08/lib/camomile/default_config -I /home/opam/.opam/4.08/lib/camomile/dyn -I /home/opam/.opam/4.08/lib/camomile/lib_default -I /home/opam/.opam/4.08/lib/camomile/library -I /home/opam/.opam/4.08/lib/cmdliner -I /home/opam/.opam/4.08/lib/gen -I /home/opam/.opam/4.08/lib/menhirLib -I /home/opam/.opam/4.08/lib/ocamlgraph -I /home/opam/.opam/4.08/lib/ppx_deriving/runtime -I /home/opam/.opam/4.08/lib/re -I /home/opam/.opam/4.08/lib/result -I /home/opam/.opam/4.08/lib/sedlex -I /home/opam/.opam/4.08/lib/seq -I /home/opam/.opam/4.08/lib/uchar -I /home/opam/.opam/4.08/lib/unionFind -I /home/opam/.opam/4.08/lib/visitors/runtime -I /home/opam/.opam/4.08/lib/zarith -I /home/opam/.opam/4.08/lib/zarith_stubs_js -I src/catala/.runtime.objs/byte -I src/catala/.runtime.objs/native -I src/catala/dcalc/.dcalc.objs/byte -I src/catala/dcalc/.dcalc.objs/native -I src/catala/desugared/.desugared.objs/byte -I src/catala/desugared/.desugared.objs/native -I src/catala/scopelang/.scopelang.objs/byte -I src/catala/scopelang/.scopelang.objs/native -I src/catala/utils/.utils.objs/byte -I src/catala/utils/.utils.objs/native -intf-suffix .ml -no-alias-deps -open Surface -o src/catala/surface/.surface.objs/byte/surface__Parser_driver.cmo -c -impl src/catala/surface/parser_driver.pp.ml)
# File "src/catala/surface/parser_driver.ml", line 229, characters 53-66:
# 229 |     | Contents contents -> (Sedlexing.Utf8.from_gen (Gen.of_string contents), None)
#                                                            ^^^^^^^^^^^^^
# Error: Unbound value Gen.of_string
#     ocamlopt src/catala/surface/.surface.objs/native/surface__Parser_driver.{cmx,o} (exit 2)
# (cd _build/default && /home/opam/.opam/4.08/bin/ocamlopt.opt -w -40 -g -I src/catala/surface/.surface.objs/byte -I src/catala/surface/.surface.objs/native -I /home/opam/.opam/4.08/lib/ANSITerminal -I /home/opam/.opam/4.08/lib/bindlib -I /home/opam/.opam/4.08/lib/bytes -I /home/opam/.opam/4.08/lib/calendar -I /home/opam/.opam/4.08/lib/camomile -I /home/opam/.opam/4.08/lib/camomile/default_config -I /home/opam/.opam/4.08/lib/camomile/dyn -I /home/opam/.opam/4.08/lib/camomile/lib_default -I /home/opam/.opam/4.08/lib/camomile/library -I /home/opam/.opam/4.08/lib/cmdliner -I /home/opam/.opam/4.08/lib/gen -I /home/opam/.opam/4.08/lib/menhirLib -I /home/opam/.opam/4.08/lib/ocamlgraph -I /home/opam/.opam/4.08/lib/ppx_deriving/runtime -I /home/opam/.opam/4.08/lib/re -I /home/opam/.opam/4.08/lib/result -I /home/opam/.opam/4.08/lib/sedlex -I /home/opam/.opam/4.08/lib/seq -I /home/opam/.opam/4.08/lib/uchar -I /home/opam/.opam/4.08/lib/unionFind -I /home/opam/.opam/4.08/lib/visitors/runtime -I /home/opam/.opam/4.08/lib/zarith -I /home/opam/.opam/4.08/lib/zarith_stubs_js -I src/catala/.runtime.objs/byte -I src/catala/.runtime.objs/native -I src/catala/dcalc/.dcalc.objs/byte -I src/catala/dcalc/.dcalc.objs/native -I src/catala/desugared/.desugared.objs/byte -I src/catala/desugared/.desugared.objs/native -I src/catala/scopelang/.scopelang.objs/byte -I src/catala/scopelang/.scopelang.objs/native -I src/catala/utils/.utils.objs/byte -I src/catala/utils/.utils.objs/native -intf-suffix .ml -no-alias-deps -open Surface -o src/catala/surface/.surface.objs/native/surface__Parser_driver.cmx -c -impl src/catala/surface/parser_driver.pp.ml)
# File "src/catala/surface/parser_driver.ml", line 229, characters 53-66:
# 229 |     | Contents contents -> (Sedlexing.Utf8.from_gen (Gen.of_string contents), None)
#                                                            ^^^^^^^^^^^^^
# Error: Unbound value Gen.of_string
```

Reply:
This is transitively used from sedlex, but since it is used directly I think it is better to explicitly depend on it instead of adding a conflict

Reply:
Thanks for figuring this out @mseri!